src/CMakeLists.txt

```diff
@@ -6,13 +6,15 @@ include(${UMF_CMAKE_SOURCE_DIR}/cmake/helpers.cmake)
 
 set(UMF_LEVEL_ZERO_INCLUDE_DIR
     ""
-    CACHE FILEPATH "Directory containing the Level Zero Headers")
```

Comment:
It's not a part of the comment's commit.

Reply:
this commit contains two small changes, both described in the commit msg. They are not relevant to the major change here, which is bumping the version.

Reply:
I mean, the change from "FILEPATH" to "FILE" is not just an update to the comment, as the commit message says. It can be confusing if something goes wrong after the commit.
Regardless, let's leave it as is.